tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -116,10 +116,11 @@ return [
 
 $array = [
     'type' => [],
-    'config' => [
-        // comments in nested arrays break things
-        'key' => 1
-    ],
+    'config' =>
+        [
```

Comment:
Should be https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACAJAQwE7cwT1QF5VgAdKVKqgchnwAc4alUBtAXQBoLrUbIUAGYBLAOYtSFXn1QzZ-ANZx8NVKwCMZGLPmoAvj0p8amMGDgN4AE0mdtFfQG5tILiAhWR0AM7JQONgQAO4ACjgIfiiYAG4QItZuIABGeGDKMADKDGYiUGLIMNgArnDueT5w2DCheGIAtpjIQpgANpXuAFY+AB4AQmkZmZj1cAAyeXDNbR0gOdiV2MgpmMn4rdBJDNh5MADqCTAAFsgAHAAM7tsQlXt4DMvbcIsxU+7YcACOxSIftZgNJpIFrtMogSr1ESFEpgnx5MStOAARWKEHg01B7hgqwO1mOyAATFi8CJWvCAMIQeqNZZQaBvEDFSoAFVWURBlX0+iAA

Please create issue around this problem